Fix the /mf force sub-commands producing an internal error on 1.17.

**Console Output**
[22:09:56 ERROR]: null
org.bukkit.command.CommandException: Unhandled exception executing command 'mf' in plugin MedievalFactions v4.2-beta-2
	at org.bukkit.command.PluginCommand.execute(PluginCommand.java:47) ~[spigot-1.17.jar:3169-Spigot-a483d2c-07ed44a]
	at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:149) ~[spigot-1.17.jar:3169-Spigot-a483d2c-07ed44a]
	at org.bukkit.craftbukkit.v1_17_R1.CraftServer.dispatchCommand(CraftServer.java:760) ~[spigot-1.17.jar:3169-Spigot-a483d2c-07ed44a]
	at net.minecraft.server.network.PlayerConnection.handleCommand(PlayerConnection.java:1944) ~[spigot-1.17.jar:3169-Spigot-a483d2c-07ed44a]
	at net.minecraft.server.network.PlayerConnection.a(PlayerConnection.java:1783) ~[spigot-1.17.jar:3169-Spigot-a483d2c-07ed44a]
	at net.minecraft.server.network.PlayerConnection.a(PlayerConnection.java:1764) ~[spigot-1.17.jar:3169-Spigot-a483d2c-07ed44a]
	at net.minecraft.network.protocol.game.PacketPlayInChat.a(PacketPlayInChat.java:46) ~[spigot-1.17.jar:3169-Spigot-a483d2c-07ed44a]
	at net.minecraft.network.protocol.game.PacketPlayInChat.a(PacketPlayInChat.java:1) ~[spigot-1.17.jar:3169-Spigot-a483d2c-07ed44a]
	at net.minecraft.network.protocol.PlayerConnectionUtils.lambda$0(PlayerConnectionUtils.java:30) ~[spigot-1.17.jar:3169-Spigot-a483d2c-07ed44a]
	at net.minecraft.server.TickTask.run(SourceFile:18) ~[spigot-1.17.jar:3169-Spigot-a483d2c-07ed44a]
	at net.minecraft.util.thread.IAsyncTaskHandler.executeTask(SourceFile:151) ~[spigot-1.17.jar:3169-Spigot-a483d2c-07ed44a]
	at net.minecraft.util.thread.IAsyncTaskHandlerReentrant.executeTask(SourceFile:23) ~[spigot-1.17.jar:3169-Spigot-a483d2c-07ed44a]
	at net.minecraft.util.thread.IAsyncTaskHandler.executeNext(SourceFile:125) ~[spigot-1.17.jar:3169-Spigot-a483d2c-07ed44a]
	at net.minecraft.server.MinecraftServer.bg(MinecraftServer.java:1125) ~[spigot-1.17.jar:3169-Spigot-a483d2c-07ed44a]
	at net.minecraft.server.MinecraftServer.executeNext(MinecraftServer.java:1118) ~[spigot-1.17.jar:3169-Spigot-a483d2c-07ed44a]
	at net.minecraft.util.thread.IAsyncTaskHandler.awaitTasks(SourceFile:134) ~[spigot-1.17.jar:3169-Spigot-a483d2c-07ed44a]
	at net.minecraft.server.MinecraftServer.sleepForTick(MinecraftServer.java:1102) ~[spigot-1.17.jar:3169-Spigot-a483d2c-07ed44a]
	at net.minecraft.server.MinecraftServer.x(MinecraftServer.java:1031) ~[spigot-1.17.jar:3169-Spigot-a483d2c-07ed44a]
	at net.minecraft.server.MinecraftServer.lambda$0(MinecraftServer.java:307) ~[spigot-1.17.jar:3169-Spigot-a483d2c-07ed44a]
	at java.lang.Thread.run(Thread.java:831) [?:?]
Caused by: java.lang.ClassCastException: class [Ljava.lang.Object; cannot be cast to class [Ljava.lang.String; ([Ljava.lang.Object; and [Ljava.lang.String; are in module java.base of loader 'bootstrap')
	at dansplugins.factionsystem.commands.ForceCommand.execute(ForceCommand.java:65) ~[?:?]
	at dansplugins.factionsystem.commands.abs.SubCommand.performCommand(SubCommand.java:146) ~[?:?]
	at dansplugins.factionsystem.CommandInterpreter.interpretCommand(CommandInterpreter.java:61) ~[?:?]
	at dansplugins.factionsystem.MedievalFactions.onCommand(MedievalFactions.java:81) ~[?:?]
	at org.bukkit.command.PluginCommand.execute(PluginCommand.java:45) ~[spigot-1.17.jar:3169-Spigot-a483d2c-07ed44a]
	... 19 more

--- a/src/main/java/dansplugins/factionsystem/commands/ForceCommand.java
+++ b/src/main/java/dansplugins/factionsystem/commands/ForceCommand.java
@@ -60,13 +60,15 @@ public class ForceCommand extends SubCommand {
      */
     @Override
     public void execute(CommandSender sender, String[] args, String key) {
-        if (!(args.length <= 0)) {
-            for (Map.Entry<List<String>, String> entry : subMap.entrySet()) {
+        if (!(args.length <= 0)) { // If the Argument has Arguments in the 'args' list.
+            for (Map.Entry<List<String>, String> entry : subMap.entrySet()) { // Loop through the SubCommands.
+                // Map.Entry<List<String>, String> example => ([Save, CMDForceSave (translation key)], forceSave)
                 try {
-                    if (safeEquals(false, args[0], entry.getKey().toArray(new String[0]))) {
+                    if (safeEquals(false, args[0], entry.getKey().toArray(new String[0]))) { // Do any of the Keys for the SubCommand match the Given Argument at index '0'.
                         try {
-                            Method method = getClass().getDeclaredMethod(entry.getValue(), CommandSender.class, String[].class);
-                            method.invoke(this, sender, args);
+                            Method method = getClass().getDeclaredMethod(entry.getValue(), CommandSender.class, String[].class); // Get the Declared method for that SubCommand.
+                            method.invoke(this, sender, args);  // Use reflection to invoke the command.
+                                                                // Due to the nature of the force-command, it is safe to use reflection here.
                         } catch (ReflectiveOperationException ex) {
                             System.out.println("DEBUG: Failed to resolve method from '" + args[0] + "'!");
                         }
@@ -77,6 +79,7 @@ public class ForceCommand extends SubCommand {
                 }
             }
         }
+        // Print out these messages if the command either isn't found or if an error occurs or if the arguments ('args') list is empty.
         sender.sendMessage(translate("&b" + getText("SubCommands")));
         Arrays.stream(commands).forEach(str -> sender.sendMessage(translate("&b" + getText("HelpForce" + str))));
     }

--- a/src/main/java/dansplugins/factionsystem/commands/ForceCommand.java
+++ b/src/main/java/dansplugins/factionsystem/commands/ForceCommand.java
@@ -62,14 +62,18 @@ public class ForceCommand extends SubCommand {
     public void execute(CommandSender sender, String[] args, String key) {
         if (!(args.length <= 0)) {
             for (Map.Entry<List<String>, String> entry : subMap.entrySet()) {
-                if (safeEquals(false, args[0], (String[]) entry.getKey().toArray())) {
-                    try {
-                        Method method = getClass().getDeclaredMethod(entry.getValue(), CommandSender.class, String[].class);
-                        method.invoke(this, sender, args);
-                    } catch (ReflectiveOperationException ex) {
-                        System.out.println("DEBUG: Failed to resolve method from '" + args[0] + "'!");
+                try {
+                    if (safeEquals(false, args[0], entry.getKey().toArray(new String[0]))) {
+                        try {
+                            Method method = getClass().getDeclaredMethod(entry.getValue(), CommandSender.class, String[].class);
+                            method.invoke(this, sender, args);
+                        } catch (ReflectiveOperationException ex) {
+                            System.out.println("DEBUG: Failed to resolve method from '" + args[0] + "'!");
+                        }
+                        return;
                     }
-                    return;
+                } catch (Exception e) {
+                    System.out.println("DEBUG: Failed to use safeEquals to determine the command chosen.");
                 }
             }
         }

